src/libraries/System.IO.Compression/src/System/IO/Compression/ZipArchiveEntry.cs

```diff
@@ -278,13 +297,23 @@ public void Delete()
             UnloadStreams();
         }
 
-        /// <summary>
-        /// Opens the entry. If the archive that the entry belongs to was opened in Read mode, the returned stream will be readable, and it may or may not be seekable. If Create mode, the returned stream will be writable and not seekable. If Update mode, the returned stream will be readable, writable, seekable, and support SetLength.
-        /// </summary>
-        /// <returns>A Stream that represents the contents of the entry.</returns>
-        /// <exception cref="IOException">The entry is already currently open for writing. -or- The entry has been deleted from the archive. -or- The archive that this entry belongs to was opened in ZipArchiveMode.Create, and this entry has already been written to once.</exception>
-        /// <exception cref="InvalidDataException">The entry is missing from the archive or is corrupt and cannot be read. -or- The entry has been compressed using a compression method that is not supported.</exception>
-        /// <exception cref="ObjectDisposedException">The ZipArchive that this entry belongs to has been disposed.</exception>
+        /// <summary>Opens the entry from the zip archive.</summary>
+        /// <returns>The stream that represents the contents of the entry.</returns>
+        /// <remarks>You use this method to access the stream for an entry in a zip archive. After retrieving the stream, you can read from or write to the stream. When you write to the stream, the modifications you make to the entry will appear in the zip archive.</remarks>
+        /// <example>The following example shows how to create a new entry, open it with the <see cref="System.IO.Compression.ZipArchiveEntry.Open" /> method, and write to the stream.
+        /// <format type="text/markdown"><![CDATA[
+        /// [!code-csharp[System.IO.Compression.ZipArchiveMode#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.io.compression.ziparchivemode/cs/program1.cs#1)]
+        /// [!code-vb[System.IO.Compression.ZipArchiveMode#1](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.io.compression.ziparchivemode/vb/program1.vb#1)]
+        /// ]]></format></example>
+        /// <exception cref="System.IO.IOException">The entry is already currently open for writing.
+        /// -or-
+        /// The entry has been deleted from the archive.
+        /// -or-
+        /// The archive for this entry was opened with the <see cref="System.IO.Compression.ZipArchiveMode.Create" /> mode, and this entry has already been written to.</exception>
+        /// <exception cref="System.IO.InvalidDataException">The entry is either missing from the archive or is corrupt and cannot be read.
+        /// -or-
+        /// The entry has been compressed by using a compression method that is not supported.</exception>
```

Comment:
All the required `<para>` elements are missing

```suggestion
        /// <exception cref="System.IO.IOException"><para>The entry is already currently open for writing.</para>
        /// <para>-or-</para>
        /// <para>The entry has been deleted from the archive.</para>
        /// <para>-or-</para>
        /// <para>The archive for this entry was opened with the <see cref="System.IO.Compression.ZipArchiveMode.Create" /> mode, and this entry has already been written to.</para></exception>
        /// <exception cref="System.IO.InvalidDataException"><para>The entry is either missing from the archive or is corrupt and cannot be read.</para>
        /// <para>-or-</para>
        /// <para>The entry has been compressed by using a compression method that is not supported.</para></exception>
```